src/main/java/seedu/billboard/logic/commands/UndoCommand.java

```diff
@@ -29,6 +29,7 @@ public CommandResult execute(Model model) throws CommandException {
         Model undoModel = VersionedBillboard.getUndoModel();
         String undoCmd = VersionedBillboard.getUndoCmd();
         model.setModel(undoModel);
-        return new CommandResult(String.format(MESSAGE_UNDO_SUCCESS, undoCmd));
+        return new CommandResult(String.format(MESSAGE_UNDO_SUCCESS, undoCmd),
+                false, false, "");
```

Comment:
maybe for `undo` commands don't switch to default list as doing so may lead to undesired behavior in some cases:
e.g. user deletes/reverts an archive expense from an archive with the archive list displayed, and then executes `undo` -> supposed to maintain view of archive list but `undo` switches it back to default list

Reply:
Agreed for the `DEFAULT_LIST`. Modified some code to try to get undoing/redoing to switch to whatever list view was displayed for that command.